### DISCUSSION TITLE
Updated firefly to V2.0.3 and wallet value went from $48 to $0.048 WHY?

### DISCUSSION THREAD

**Discussion by mattayre01** [iotaledger/firefly#7761] [Help & Support] [answered]
After updating wallet to latest version my balance dropped from $48 to $0.048. Also assembly and shimmer has issues! Anyone else had this problem or knows what went wrong??

**Comment by begonaalvarezd** [comment, marked as answer]
To fix the missing balances, just run the balance finder from settings > advanced > balance finder. Mark the "search in current wallet only" if you want to optimize the search for the selected wallet

Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.3 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app because your rewards are in the shimmer network, not the iota network. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

**Reply by mattayre01** [reply]
Hi, thanks for the reply. My IOTA returned once running the finer feature. Yes I don't see any assembly or shimmer. I note your reply on those tokens and will check it out.
Thanks.

**Comment by Meisi21** [comment]
....to fix missing balances IOTA. What if the missing amount is not found ? Which other possibility do I have ?

**Reply by begonaalvarezd** [reply]
https://github.com/iotaledger/firefly/issues/7768#issuecomment-1845286126
